client/src/common/file.ts

```diff
@@ -78,11 +78,11 @@ function size(bytes: number, system: [number, string][]): string {
 
 export function formatFileSize(bytesize: number): string {
   const SYSTEM: [number, string][] = [
-    [Math.pow(1024, 0), translate('common.filesize.bytes')],
-    [Math.pow(1024, 1), translate('common.filesize.kilobytes')],
-    [Math.pow(1024, 2), translate('common.filesize.megabytes')],
-    [Math.pow(1024, 3), translate('common.filesize.gigabytes')],
-    [Math.pow(1024, 4), translate('common.filesize.terabytes')],
+    [Math.pow(1024, 0), msTranslate('common.filesize.bytes')],
```

Comment:
Should be `const SYSTEM: [number, Translatable][]` instead, we're not translating directly.